.idea/gradle.xml

```diff
@@ -23,4 +23,4 @@
       </GradleProjectSettings>
     </option>
   </component>
-</project>
+</project>
```

Comment:
This error seems irritating, I'll add this file to `.gitignore` but for now any suggestions to fix this?

Reply:
@prayutsu you can use github and edit this file in your branch

Reply:
is it a good practice to edit this from here?